operator/apis/loki/v1beta1/alertingrule_defaulting_webhook.go

```diff
@@ -0,0 +1,27 @@
+package v1beta1
+
+import (
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+const tenantIDLabelname = "tenantId"
+
+var _ webhook.Defaulter = &AlertingRule{}
```

Comment:
If we stick to the webhook, can we move the webhook code out of the API package similar to the validating webhooks? I don't know why the operator-sdk defaults to attaching the webhook code to the API types.